src/System.Net.HttpListener/src/System/Net/Managed/HttpListenerResponse.Managed.cs

```diff
@@ -315,7 +315,15 @@ private static string FormatHeaders(WebHeaderCollection headers)
                     {
                         if (anyValues)
                         {
-                            sb.Append(", ");
+                            if (key.Equals(HttpKnownHeaderNames.SetCookie, StringComparison.InvariantCultureIgnoreCase) ||
+                                key.Equals(HttpKnownHeaderNames.SetCookie2, StringComparison.InvariantCultureIgnoreCase))
```

Comment:
It's possible that `StringComparison.OrdinalIgnoreCase` would be better (and faster) here.

@stephentoub PTAL

Reply:
Yeah, let's use ordinal ignore case.